test/test_unloader.py

```diff
@@ -3,6 +3,8 @@
 import tempfile
 import unittest
 
+from mock import mock
```

Comment:
```suggestion
from mock import mock

```
Could do with a line break to divide the 3rd-party modules from our own.